macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql

```diff
@@ -10,7 +10,7 @@
             edr_bucket_start,
             edr_bucket_end,
             1 as joiner
-          from ({{ elementary.complete_buckets_cte(time_bucket) }})
+          from ({{ elementary.complete_buckets_cte(time_bucket) }}) rs
```

Comment:
What does "rs" stand for?

Reply:
Result-set.
I was also unaware at first, the naming is a community contribution.
Should we change it to `results`?

Reply:
Decided to rename it.

Reply:
He called it result set -> rs . 
I would prefer to use a name that describes what the result set is, makes the code more readable